src/ingress.ts

```diff
@@ -152,7 +176,8 @@ export class IngressV1Beta1 extends Resource {
 
       const host = rule.host ?? '';
       const backend = rule.backend;
-      const path = rule.path;
+      const path = rule.path ?? '/';
+      const pathType = rule.pathType ?? HttpIngressPathType.PREFIX;
```

Comment:
In the raw spec, "path" is marked as an optional field -- but it is required if the pathType is either "Path" or "Prefix". Our `addDefaultBackend` and `addHostDefaultBackend` APIs don't require the user to provide any path, but I think "/" is the appropriate default here.